src/main/java/frc/robot/subsystems/swerve/module/Modules.java

```diff
@@ -0,0 +1,154 @@
+package frc.robot.subsystems.swerve.module;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.kinematics.SwerveModulePosition;
+import edu.wpi.first.math.kinematics.SwerveModuleState;
+import frc.robot.constants.MathConstants;
+import frc.robot.subsystems.GBSubsystem;
+import org.littletonrobotics.junction.Logger;
+
+import java.util.Arrays;
+
+public class Modules extends GBSubsystem {
+
+	private final Module[] modules;
+	private final ModulesCommandsBuilder commandsBuilder;
+
+	public Modules(String logPath, Module... modules) {
+		super(logPath + ModuleConstants.LOG_PATH_ADDITION);
+		this.modules = modules;
+		this.commandsBuilder = new ModulesCommandsBuilder(this);
+	}
+
+	public Module getModule(ModuleUtils.ModulePosition modulePosition) {
+		return modules[modulePosition.getIndex()];
+	}
+
+	public ModulesCommandsBuilder getCommandsBuilder() {
+		return commandsBuilder;
+	}
+
+	public void updateInputs() {
+		for (Module currentModule : modules) {
+			currentModule.updateInputs();
+		}
+		Logger.recordOutput(getLogPath() + "CurrentStates", getCurrentStates());
+		Logger.recordOutput(getLogPath() + "TargetStates", getTargetStates());
+	}
+
+
+	public void resetModulesAngleByEncoder() {
+		for (Module module : modules) {
+			module.resetByEncoder();
+		}
+	}
+
+	public void setBrake(boolean brake) {
+		for (Module currentModule : modules) {
+			currentModule.setBrake(brake);
+		}
+	}
+
+
+	protected void pointWheels(Rotation2d targetSteerPosition, boolean optimize) {
+		for (Module module : modules) {
+			module.pointSteer(targetSteerPosition, optimize);
+		}
+	}
+
+	protected void pointWheelsInCircle() {
+		boolean optimizeAngle = true;
```

Comment:
Why is this boolean here? Shouldn't you want to get it as a parameter in the function?

Reply:
If u want the wheels to be in a circle u will want the optimize for sure